TACC/protx
# Task/cooks 165 update instructions

## Overview: ##

Major refactor if the way Report Instructions are displayed across all report views. 

## Related Jira tickets: ##

* [COOKS-165](https://jira.tacc.utexas.edu/browse/COOKS-165)

## Summary of Changes: ##

Centralizes all instructions in a single component that takes a single string parameter (the instructions to display) as an argument. Can be customized per report.

## Testing Steps: ##

1. Build project.
2. View any report view.
3. Check for instructions at first load per report view

## UI Photos:

![Screen Shot 2021-12-01 at 12 15 51 AM](https://user-images.githubusercontent.com/3238078/144182288-6654734b-c685-430c-8516-a01d0c45bdd6.png)

![Screen Shot 2021-12-01 at 12 16 02 AM](https://user-images.githubusercontent.com/3238078/144182301-feed889c-6a27-4cfb-8009-e910b9cb636c.png)

![Screen Shot 2021-12-01 at 12 16 51 AM](https://user-images.githubusercontent.com/3238078/144182331-c887cbaf-dd6a-496d-935e-aa5db3cf7779.png)

![Screen Shot 2021-12-01 at 12 17 09 AM](https://user-images.githubusercontent.com/3238078/144182365-be147f62-06b4-4827-8314-44a0c96765e7.png)

![Screen Shot 2021-12-01 at 12 18 24 AM](https://user-images.githubusercontent.com/3238078/144182398-164f021f-603d-4b07-b64d-e94679e026b6.png)

![Screen Shot 2021-12-01 at 12 18 46 AM](https://user-images.githubusercontent.com/3238078/144182419-50a1e6e3-3a4b-4b91-b6c9-418817c4bc08.png)

## Notes: ##

None.
